G1ANT.Addon.GoogleDocs/G1ANT.Addon.GoogleDocs.Tests/Commands/googlesheet.close/ShouldCloseSheet_WhenCloseingSheet.G1ANT

```diff
@@ -0,0 +1,8 @@
+﻿addon googledocs version 2.205.17304.1000
+addon core version 4.99.18340.1354
+addon language version 4.99.18340.1354
+
+♥FileID = ‴147EH2vEjGVtbzzkT6XaI0eNZlY5Ec91wlvxN3HC4GMc‴
+googlesheet.open ♥FileID result ♥sheetHandle
```

Comment:
Inconsequent variable name casing (once first letter is upper case, once it's lower cased). Is there a reason for it?